Tasks/Maven/task.json

```diff
@@ -80,12 +93,23 @@
             "groupName":"advanced",
             "defaultValue":"default",
             "helpMarkDown": "Will attempt to discover the path to the selected JDK version and set JAVA_HOME accordingly.",
+            "visibleRule":"javaHomeSelection = JDKVersion",
             "options": {
                 "default":"default",
                 "1.8":"JDK 8",
                 "1.7":"JDK 7",
                 "1.6":"JDK 6"
             }
+        },     
+        {
+            "name":"jdkUserInputPath",
+            "type":"string",
+            "label":"JDK Path",
+            "required":true,
+            "groupName":"advanced",
+            "defaultValue":"",
+            "helpMarkDown": "Sets JAVA_HOME to the given path.",
+            "visibleRule":"javaHomeSelection = Path"
         },
         { 
             "name": "jdkArchitecture", 
```

Comment:
Yes if you build the tasks, it will generate the files. I have a batch file in my PATH that I use to build from the root of the tasks repo:

call npm install gulp -g
call npm install
call gulp